pfio/v2/local.py

```diff
@@ -158,3 +158,6 @@ def glob(self, pattern: str):
         return [
             str(item.relative_to(self.cwd))
             for item in pathlib.Path(self.cwd).glob(pattern)]
+
+    def normpath(self, file_path: str) -> str:
+        return "local" + os.path.normpath(os.path.join(self.cwd, file_path))
```

Comment:
`Local` supports relative path. That said, `os.path.join(..)` here can be anything such as `foobar` or `/foo/bar/baz` or `/foo/bar/.././boom/../../../../`. `/foo/bar/../baz` and `/foo/baz` should be treated as same, aren't they?

Reply:
I think so too, therefore I normalize its path by calling `os.path.normpath()`.

```
Python 3.9.16 (main, Feb 20 2023, 13:34:31) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> os.path.normpath("/foo/bar/../baz")
'/foo/baz'
>>> os.path.normpath("/foo/baz")
'/foo/baz'
>>> os.path.normpath("/foo/bar/../baz") == os.path.normpath("/foo/baz")
True
```